Casks/usr-sse2-rdm.rb

```diff
@@ -0,0 +1,16 @@
+cask "usr-sse2-rdm" do
+  version "2.5.0"
+  sha256 "9d583e43742018e45bad79c8af9e0accec6b6a10d94c60ac7d31721bbb3c237e"
+
+  url "https://github.com/usr-sse2/RDM/releases/download/#{version}/RDM.zip"
+  name "RDM"
+  desc "Set a Retina display to custom resolutions"
+  homepage "https://github.com/usr-sse2/RDM"
+
+  depends_on macos: ">= :sierra"
+
+  app "RDM.app"
+
+  uninstall quit:    "RDM",
```

Comment:
Not that important but using the more specific form can only improve things I guess
```suggestion
  uninstall quit:    "net.alkalay.RDM",
```